js/packages/web/src/views/artCreate/infoStep.tsx

```diff
@@ -17,6 +18,8 @@ export const InfoStep = (props: {
   );
   const [form] = Form.useForm();
 
+  const [values, setValues] = useState({} as IMetadataExtension);
```

Comment:
```suggestion
  const [values, setValues] = useState<IMetadataExtension>({});
```

For the future this is the more idiomatic approach.

Reply:
Awesome, thank you

Reply:
I suppose it should actually be
```const [values, setValues] = useState<IMetadataExtension>(props.attributes);```